l10n_es_aeat_partner_check/__manifest__.py

```diff
@@ -6,9 +6,9 @@
     "author": "Acysos S.L., Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/l10n-spain",
     "category": "Accounting",
-    "version": "16.0.1.0.1",
+    "version": "17.0.1.0.1",
     "license": "AGPL-3",
-    "depends": ["base", "base_vat", "l10n_es_aeat"],
+    "depends": ["base", "base_vat", "l10n_es_aeat", "l10n_es_edi_facturae"],
```

Comment:
Why this dependancy is needed?

Reply:
Is no needed, sorry. Thanks for your remark!